ui/css/oae.signup.css

```diff
@@ -28,38 +36,142 @@ main > h3 {
     padding-bottom: 10px;
 }
 
+#signup-institution-container,
+#signup-options-container {
+    margin: 0 20px;
+    padding: 40px;
+}
+
+#signup-institution,
+#signup-options {
+    margin: auto;
+    max-width: 992px;
+}
+
+
+/**************************
+ ** INSTITUTIONAL SEARCH **
+ **************************/
+
+/** Layout **/
+
+#signup-institution-container {
+    border-bottom: solid 1px #DADADA;
+    display: none;
+}
+
+#signup-institution {
+    display: -webkit-flex;
+    display:  -ms-flexbox;
+    display:         flex;
+}
+
+#signup-institution > div {
+    /* IE10 defaults to 1 0 auto, so set this */
+    -ms-flex: 0 1 auto;
+}
+
+#signup-institution-instruction {
+    margin-right: 50px;
+    min-width: 250px;
+}
+
+#signup-institution-search {
```

Comment:
I would make the placeholder text of the search box the same color as the placeholder texts elsewhere in the page